src/main/java/com/cloudbees/jenkins/support/SupportAction.java

```diff
@@ -92,6 +104,12 @@ public class SupportAction implements RootAction, StaplerProxy {
      */
     private final Logger logger = Logger.getLogger(SupportAction.class.getName());
 
+    private static final Path SUPPORT_BUNDLE_CREATION_FOLDER =
+            Paths.get(System.getProperty("java.io.tmpdir")).resolve("support-bundle");
+    public static final String SYNC_SUPPORT_BUNDLE = "support-bundle.zip";
+
+    private static final Map<UUID, SupportBundleAsyncGenerator> generatorByTaskId = new ConcurrentHashMap<>();
```

Comment:
```suggestion
    private final Map<UUID, SupportBundleAsyncGenerator> generatorByTaskId = new ConcurrentHashMap<>();
```
(it is a field of a singleton, so still just one, but works better with some `JenkinsRule` tests)

Reply:
I have removed it for generatorByTaskId

but im using a few static fileds like 
```
private static final Path SUPPORT_BUNDLE_CREATION_FOLDER =
        Paths.get(System.getProperty("java.io.tmpdir")).resolve("support-bundle");
public static final String SYNC_SUPPORT_BUNDLE = "support-bundle.zip";
  ```
  
  which are being used in other classes

Reply:
Sure but those are not mutable state.